# Russian translation for linuxmint
# Copyright (c) 2009 Rosetta Contributors and Canonical Ltd 2009
# This file is distributed under the same license as the linuxmint package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2009.
#
msgid ""
msgstr ""
"Project-Id-Version: linuxmint\n"
"Report-Msgid-Bugs-To: FULL NAME <EMAIL@ADDRESS>\n"
"POT-Creation-Date: 2014-04-14 12:43+0100\n"
"PO-Revision-Date: 2014-05-17 21:09+0000\n"
"Last-Translator: removed <Unknown>\n"
"Language-Team: Russian <ru@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2015-06-02 10:09+0000\n"
"X-Generator: Launchpad (build 17532)\n"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:166
msgid "Size mismatch"
msgstr "При несовпадении размера"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:167
msgid "Modification time mismatch"
msgstr "При несовпадении времени изменения"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:168
msgid "Checksum mismatch"
msgstr "При несовпадении контрольной суммы"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:276
msgid "Backup or Restore"
msgstr "Резервирование и Восстановление"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:301
msgid "Output:"
msgstr "Формат вывода:"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:376
msgid "Software selection:"
msgstr "Файл списка:"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:1053
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1077
msgid "(Not created with the Backup Tool)"
msgstr "(Не создан при помощи средства \"Резервное копирование\")"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:1527
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1570
msgid "An error occurred while accessing the file"
msgstr "Ошибка доступа к файлу"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:34
#: usr/lib/linuxmint/mintBackup/mintBackup.py:99
#: usr/lib/linuxmint/mintBackup/mintBackup.py:238
#: usr/lib/linuxmint/mintBackup/mintBackup.py:427
#: usr/lib/linuxmint/mintBackup/mintBackup.py:443
#: usr/lib/linuxmint/mintBackup/mintBackup.py:501
#: usr/lib/linuxmint/mintBackup/mintBackup.py:504
#: usr/lib/linuxmint/mintBackup/mintBackup.py:556
#: usr/lib/linuxmint/mintBackup/mintBackup.py:559
#: usr/lib/linuxmint/mintBackup/mintBackup.py:579
#: usr/lib/linuxmint/mintBackup/mintBackup.py:602
#: usr/lib/linuxmint/mintBackup/mintBackup.py:609
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1426
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1519
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1527
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1570
msgid "Backup Tool"
msgstr "Резервное копирование"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:35
msgid "Make a backup of your home directory"
msgstr "Создание резервной копии вашей домашней директории"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:155
msgid "Preserve structure"
msgstr "Сохранить структуру (без сжатия)"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:157
msgid ".tar file"
msgstr ".tar архив"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:158
msgid ".tar.bz2 file"
msgstr ".tar.bz2 архив"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:159
msgid ".tar.gz file"
msgstr ".tar.gz архив"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:165
msgid "Never"
msgstr "Никогда"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:169
msgid "Always"
msgstr "Всегда"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:191
msgid "Excluded paths"
msgstr "Исключенные пути"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:201
msgid "Type"
msgstr "Тип"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:205
msgid "Detail"
msgstr "Подробности"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:211
#: usr/lib/linuxmint/mintBackup/mintBackup.py:220
msgid "Path"
msgstr "Путь"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:214
#: usr/lib/linuxmint/mintBackup/mintBackup.py:223
msgid "Error"
msgstr "Ошибка"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:255
msgid "Store?"
msgstr "Архивировать?"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:258
#: usr/lib/linuxmint/mintBackup/mintBackup.py:271
msgid "Name"
msgstr "Имя"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:268
msgid "Install"
msgstr "Установить"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:277
msgid "Choose from the following options"
msgstr "Выберите нужное действие"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:279
#: usr/lib/linuxmint/mintBackup/mintBackup.py:295
#: usr/lib/linuxmint/mintBackup/mintBackup.py:309
#: usr/lib/linuxmint/mintBackup/mintBackup.py:316
#: usr/lib/linuxmint/mintBackup/mintBackup.py:320
#: usr/lib/linuxmint/mintBackup/mintBackup.py:325
msgid "Backup files"
msgstr "Резервное копирование файлов"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:280
#: usr/lib/linuxmint/mintBackup/mintBackup.py:329
#: usr/lib/linuxmint/mintBackup/mintBackup.py:339
#: usr/lib/linuxmint/mintBackup/mintBackup.py:345
#: usr/lib/linuxmint/mintBackup/mintBackup.py:350
msgid "Restore files"
msgstr "Восстановление файлов"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:281
#: usr/lib/linuxmint/mintBackup/mintBackup.py:354
#: usr/lib/linuxmint/mintBackup/mintBackup.py:359
#: usr/lib/linuxmint/mintBackup/mintBackup.py:365
#: usr/lib/linuxmint/mintBackup/mintBackup.py:370
msgid "Backup software selection"
msgstr "Резервное копирование программ"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:282
#: usr/lib/linuxmint/mintBackup/mintBackup.py:374
#: usr/lib/linuxmint/mintBackup/mintBackup.py:379
#: usr/lib/linuxmint/mintBackup/mintBackup.py:386
msgid "Restore software selection"
msgstr "Восстановление программ"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:284
msgid "Make a backup of your files"
msgstr "Создать резервную копию ваших файлов"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:285
msgid "Save the list of installed applications"
msgstr "Сохранить список установленных приложений"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:286
msgid "Restore a previous backup"
msgstr "Восстановить файлы из резервной копии"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:287
msgid "Restore previously installed applications"
msgstr "Восстановить ранее установленные приложения"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:296
msgid "Please select a source and a destination for your backup"
msgstr "Выберите источник и расположение резервной копии"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:297
#: usr/lib/linuxmint/mintBackup/mintBackup.py:333
#: usr/lib/linuxmint/mintBackup/mintBackup.py:341
msgid "Source:"
msgstr "Источник:"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:298
#: usr/lib/linuxmint/mintBackup/mintBackup.py:334
msgid "Destination:"
msgstr "Сохранить в:"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:299
#: usr/lib/linuxmint/mintBackup/mintBackup.py:335
msgid "Advanced options"
msgstr "Дополнительные настройки"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:300
#: usr/lib/linuxmint/mintBackup/mintBackup.py:342
msgid "Description:"
msgstr "Описание:"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:302
#: usr/lib/linuxmint/mintBackup/mintBackup.py:336
msgid "Overwrite:"
msgstr "Перезаписывать:"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:303
msgid "Confirm integrity"
msgstr "Проверить целостность данных"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:304
msgid "Follow symlinks"
msgstr "Следовать по символьным ссылкам"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:305
msgid "Preserve permissions"
msgstr "Сохранить права доступа"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:306
msgid "Preserve timestamps"
msgstr "Сохранить отметки времени"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:310
msgid "Please select any files or directories you want to exclude"
msgstr "Какие файлы или папки вы хотите исключить"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:311
msgid "Exclude files"
msgstr "Исключить файлы"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:312
msgid "Exclude directories"
msgstr "Исключить папки"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:313
msgid "Remove"
msgstr "Удалить"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:317
msgid "Please review the information below before starting the backup"
msgstr ""
"Проверьте указанную информацию, прежде чем начнётся резервное копирование"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:321
msgid "Please wait while your files are being backed up"
msgstr "Пожалуйста, подождите, создаётся резервная копия ваших файлов"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:322
#: usr/lib/linuxmint/mintBackup/mintBackup.py:367
msgid "Backing up:"
msgstr "Создается резервная копия:"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:326
#: usr/lib/linuxmint/mintBackup/mintBackup.py:371
msgid "The backup is now finished"
msgstr "Резервное копирование завершено"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:330
msgid ""
"Please choose the type of backup to restore, its location and a destination"
msgstr ""
"Выберите тип восстанавливаемой резервной копии, её расположение и путь для "
"распаковки"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:331
msgid "Archive"
msgstr "Архив"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:332
msgid "Directory"
msgstr "Папка"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:340
msgid "Please review the information below"
msgstr "Проверьте указанную информацию"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:346
msgid "Please wait while your files are being restored"
msgstr "Пожалуйста, подождите, ваши файлы восстанавливаются"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:347
msgid "Restoring:"
msgstr "Восстанавливается:"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:351
msgid "The restoration of the files is now finished"
msgstr "Восстановление файлов завершено"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:355
msgid "Please choose a destination"
msgstr "Пожалуйста, выберите путь"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:356
#: usr/lib/linuxmint/mintBackup/mintBackup.py:520
msgid "Destination"
msgstr "Путь назначения"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:360
msgid "The list below shows the packages you added to Linux Mint"
msgstr "Следующий список содержит пакеты, добавленные вами в Linux Mint"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:361
#: usr/lib/linuxmint/mintBackup/mintBackup.py:381
msgid "Select all"
msgstr "Выбрать всё"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:362
#: usr/lib/linuxmint/mintBackup/mintBackup.py:382
msgid "Deselect all"
msgstr "Снять выделение"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:366
msgid "Please wait while your software selection is being backed up"
msgstr "Пожалуйста, подождите, выбранные вами программы резервируются"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:375
msgid "Please select a saved software selection"
msgstr "Пожалуйста, выберите сохранённый набор программ"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:380
msgid "Select the packages you want to install"
msgstr "Выберите пакеты, которые хотите установить"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:383
msgid "Refresh"
msgstr "Обновить"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:387
msgid "The restoration is now finished"
msgstr "Восстановление завершено"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:388
msgid "Your package selection was restored succesfully"
msgstr "Ваш набор пакетов успешно восстановлен"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:436
#: usr/lib/linuxmint/mintBackup/mintBackup.py:452
msgid "Invalid path"
msgstr "Неверный путь"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:436
#: usr/lib/linuxmint/mintBackup/mintBackup.py:452
#, python-format
msgid "%s is not located within your source directory. Not added."
msgstr "%s нет в указанной папке. Добавление невозможно."

#: usr/lib/linuxmint/mintBackup/mintBackup.py:501
msgid "Please choose directories for the source and the destination"
msgstr "Пожалуйста, выберите каталог-источник и место назначения"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:504
msgid ""
"Please choose different directories for the source and the destination"
msgstr ""
"Пожалуйста, укажите другие папки в качестве источника и пути назначения"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:519
msgid "Source"
msgstr "Источник"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:522
msgid "Description"
msgstr "Описание"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:526
msgid "Compression"
msgstr "Сжатие"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:530
msgid "Overwrite destination files"
msgstr "Перезаписывать одноимённые файлы"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:533
msgid "Exclude"
msgstr "Исключить"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:556
msgid "Please choose a file to restore from"
msgstr "Пожалуйста, выберите файлы для восстановления"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:559
#: usr/lib/linuxmint/mintBackup/mintBackup.py:579
msgid "Please choose a destination directory"
msgstr "Пожалуйста, укажите папку назначения"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:602
msgid "No packages need to be installed at this time"
msgstr "На данный момент нет пакетов, которые необходимо установить"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:609
msgid "Please select one or more packages to install"
msgstr "Пожалуйста, выберите один или несколько пакетов для установки"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:666
#: usr/lib/linuxmint/mintBackup/mintBackup.py:667
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1143
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1145
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1439
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1440
msgid "Calculating..."
msgstr "Вычисление..."

#: usr/lib/linuxmint/mintBackup/mintBackup.py:716
#: usr/lib/linuxmint/mintBackup/mintBackup.py:756
msgid "Skipping broken link"
msgstr "Пропуск нерабочей ссылки"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:717
msgid "Broken link"
msgstr "Нерабочая ссылка"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:720
#: usr/lib/linuxmint/mintBackup/mintBackup.py:760
msgid "Skipping link"
msgstr "Пропуск ссылки"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:784
#: usr/lib/linuxmint/mintBackup/mintBackup.py:793
#: usr/lib/linuxmint/mintBackup/mintBackup.py:802
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1204
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1215
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1228
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1293
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1302
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1312
msgid "Skipping identical file"
msgstr "Пропуск одинаковых файлов"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:824
#, python-format
msgid ""
"Warning: Some files were not saved, copied: %(current_file)d files out of "
"%(total)d total"
msgstr ""
"Внимание: некоторые файлы не сохранены, скопировано: %(current_file)d файлов "
"из %(total)d"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:828
msgid "An error occured during the backup"
msgstr "В процессе резервного копирования произошла ошибка"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:838
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1484
msgid "The backup was aborted"
msgstr "Резервное копирование было отменено"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:846
msgid "The backup completed successfully"
msgstr "Резервное копирование успешно завершено"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:931
msgid "Checksum Mismatch:"
msgstr "Несоответствие контрольной суммы:"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:932
msgid "Checksum Mismatch"
msgstr "Несоответствие контрольной суммы"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:982
#: usr/lib/linuxmint/mintBackup/mintBackup.py:984
#: usr/lib/linuxmint/mintBackup/mintBackup.py:1011
msgid "Calculating checksum"
msgstr "Вычисление контрольной суммы"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:1336
#, python-format
msgid ""
"Warning: Some files were not restored, copied: %(current_file)d files out of "
"%(total)d total"
msgstr ""
"Предупреждение! Некоторые файлы не были восстановлены, скопированы: "
"%(current_file)d файлов из %(total)d"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:1339
msgid "An error occured during the restoration"
msgstr "Возникла ошибка в процессе восстановления"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:1350
msgid "The restoration was aborted"
msgstr "Восстановление было отменено"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:1358
msgid "The restoration completed successfully"
msgstr "Восстановление успешно завершено"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:1475
msgid "An error occured during the backup:"
msgstr "Возникла ошибка в процессе резервного копирования:"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:1492
msgid "Your software selection was backed up succesfully"
msgstr "Ваш набор программ был успешно зарезервирован"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:1519
msgid "The specified file is not a valid software selection"
msgstr "Указанный файл не является корректным файлом списка"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:1563
msgid "Could not locate the package"
msgstr "Не удалось найти пакет"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:1594
msgid "Please wait, this can take some time"
msgstr "Пожалуйста, подождите, процесс может занять некоторое время"

#: usr/lib/linuxmint/mintBackup/mintBackup.py:1596
msgid "The installation is complete"
msgstr "Установка завершена"

#~ msgid "Backup successful"
#~ msgstr "Резервное копирование успешно завершено"

#~ msgid "An error occurred during the backup:"
#~ msgstr "В процессе резервного копирования произошла ошибка:"

#~ msgid "Cleaning up..."
#~ msgstr "Очистка..."

#~ msgid "An error occurred while restoring the backup archive:"
#~ msgstr "Произошла ошибка в процессе восстановления резервной копии:"

#~ msgid "Opening the backup archive..."
#~ msgstr "Открытие резервной копии..."

#~ msgid "Read error"
#~ msgstr "Ошибка чтения"

#~ msgid "An error occurred while opening the backup:"
#~ msgstr "Произошла ошибка в процессе открытия резервной копии:"

#~ msgid "Hidden paths"
#~ msgstr "Скрытые пути"

#~ msgid "Restore"
#~ msgstr "Восстановить"

#~ msgid "View content"
#~ msgstr "Просмотр содержимого"

#~ msgid "Extracting the content of the backup into your home directory..."
#~ msgstr "Извлечение содержимого резервной копии в вашу домашнюю директорию..."

#~ msgid "Copying the backup file into your home directory..."
#~ msgstr "Копирование резервной копии в вашу домашнюю директорию..."

#~ msgid "is not located within your home directory. Not added."
#~ msgstr "не расположен в пределах вашей домашней директории. Не добавлен."

#~ msgid "Restoration successful"
#~ msgstr "Восстановление успешно завершено"

#~ msgid "Restore your personal data from this backup"
#~ msgstr "Восстановление ваших личных данных из данной резервной копии"

#~ msgid "Included"
#~ msgstr "Включенные"

#~ msgid "Overwrite existing files"
#~ msgstr "Заменить существующие файлы"

#~ msgid "Restoration failed"
#~ msgstr "Не удалось завершить восстановление"

#~ msgid "<big><b>Load data into your home directory</b></big>"
#~ msgstr "<big><b>Загрузка данных в ваш домашний каталог</b></big>"

#~ msgid "Excluded Files and Directories"
#~ msgstr "Исключенные файлы и каталоги"

#~ msgid "Exclude folders"
#~ msgstr "Исключить каталоги"

#~ msgid "Archiving your home directory..."
#~ msgstr "Архивирование вашей домашней директории..."

#~ msgid "Your home directory was successfully backed-up into"
#~ msgstr "Резервная копия вашей домашней директории успешно создана в"

#~ msgid "Backup failed"
#~ msgstr "Резервное копирование завершилось неудачей"

#~ msgid "Your backup was successfully restored"
#~ msgstr "Ваша резервная копия была успешно восстановлена"

#~ msgid "Included hidden directories"
#~ msgstr "Включенные скрытые директории"

#~ msgid "Backup"
#~ msgstr "Резервное копирование"
